packages/cli/package.json

```diff
@@ -1,22 +1,17 @@
 {
   "name": "@appland/appmap",
-  "version": "3.1.0",
+  "version": "3.0.0",
   "description": "",
-  "main": "built/cli.js",
-  "bin": "built/cli.js",
+  "main": "src/cli.js",
+  "bin": "src/cli.js",
```

Comment:
Is this a conflict with the TypeScript migration?

Reply:
@dustinbyrne What should happen here?

Reply:
Without looking at the `git log`, it looks like this branch is based off the commit that includes typescript and this commit rolls all those changes back.